cdat/cdat/DictEdit/DictEditor.py

```diff
@@ -121,18 +113,23 @@ def __init__(self):
         self.key_value_rows = []
         self.rows = QVBoxLayout()
         self.clearing = False
+        rec = QApplication.desktop().screenGeometry()
+        self.setMaximumHeight(rec.height() * .8)
 
-        button = QVBoxLayout()
         wrap = QVBoxLayout()
         add_button = QPushButton()
         add_button.setText("New Line")
         add_button.clicked.connect(self.insertRow)
-        button.addWidget(add_button)
 
-        self.setLayout(wrap)
-        wrap.addLayout(self.rows)
-        wrap.addLayout(button)
+        scroll_widget = QWidget()
```

Comment:
The scroll widget stuff should be done outside of DictEdit; DictEdit doesn't necessarily have to scroll, but in our usage in the AxisEditor we need it to do so.